enterprise/utils.py

```diff
@@ -982,8 +982,22 @@ def get_enterprise_worker_user():
     """
     Return the user object of enterprise worker user.
     """
+    return __get_service_worker(settings.ENTERPRISE_SERVICE_WORKER_USERNAME)
+
+
+def get_ecommerce_worker_user():
+    """
+    Return the user object of ecommerce worker user.
+    """
+    return __get_service_worker(settings.ECOMMERCE_SERVICE_WORKER_USERNAME)
+
+
+def __get_service_worker(service_worker_username):
```

Comment:
Still a Python newbie in general. I have read previously that starting a method with "_" or "__" is a way to denote a private method. The idea is that I wanted to deter people from calling "__get_service_worker" directly and rather use "get_ecommerce_worker_user" or the pre-existing "get_enterprise_worker_user". Let me know if I'm violating any specific convention here.

Reply:
I think this is a good way to do it, just change it from a `__` to a `_` prefix. Double-underscore ("dunder"), is usually for a class's operator overrides or constructor (e.g. `__init__`, `__gt__`, etc)